src/sections/Meshery/meshery-operator/index.js

```diff
@@ -21,7 +21,7 @@ const MesheryOperatorPage = () => {
             <div className="hero-section">
               <Row>
                 <Col lg={6} md={6} sm={12}>
-                  <img src={Operator} className="logo" />
+                <StaticImage src="../../../assets/images/meshery-operator/meshery-operator.png" className="logo" />
```

Comment:
One change, in order to keep the code clean.

After all the imports,  assign the image path which is going be used in the static image component to a variable like
`const Operator="enter the image path here"` and then use this variable name as the `src` value in `StaticImage` component.

Reply:
Make this update at other places as well in the PR, where you are using StaticImage.